### PR TITLE
PSMDB-1369 Added Telemetry

### PR DESCRIPTION
**Pull request by nastena1606**
modified:   docs/404.md
	modified:   docs/install/apt.md
	modified:   docs/install/docker.md
	modified:   docs/install/index.md
	modified:   docs/install/yum.md
	new file:   docs/telemetry.md
	modified:   mkdocs-base.yml

### CODE REVIEW

File: docs/install/docker.md [hunk 1]
```diff
@@ -10,7 +10,13 @@ For more information about using Docker, see the [Docker Docs](https://docs.dock
 
     By default, Docker will pull the image from Docker Hub if it is not available locally.
 
+<<<<<<< HEAD
```
Comment (sandraromanchenko) [inline docs/install/docker.md]:
```suggestion
<<<<<<< HEAD
```
It seems a conflict, please resolve.

File: docs/telemetry.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+"metrics":
+[{"key" : "deployment","value" : "PACKAGE"},
+{"key" : "pillar_version","value" : "7.0.3-2"},
+{"key" : "OS","value" : "Oracle Linux Server 8.8"},
+{"key" : "hardware_arch","value" : "x86_64 x86_64"}]}]
+```
+
+## Disable telemetry
+
+Starting with Percona Server for MongoDB 5.0.22-19, telemetry is enabled by default. If you decide not to send usage data to Percona, you can set the `PERCONA_TELEMETRY_DISABLE=1` environment variable for either the root user or in the operating system prior to the installation process.
```
Comment (sandraromanchenko) [inline docs/telemetry.md]:
```suggestion
Starting with Percona Server for MongoDB 7.0.4-3, telemetry is enabled by default. If you decide not to send usage data to Percona, you can set the `PERCONA_TELEMETRY_DISABLE=1` environment variable for either the root user or in the operating system prior to the installation process.
```

**Review by sandraromanchenko — CHANGES_REQUESTED** [review]

File: docs/install/docker.md (hunk 1 above)
Comment (nastena1606) [reply docs/install/docker.md]: Fixed

File: docs/telemetry.md (hunk 2 above)
Comment (nastena1606) [reply docs/telemetry.md]: Updated

**Review by sandraromanchenko — APPROVED** [review]

**Pull request merged**